Packs/EWS/Integrations/EWSO365/README.md

```diff
@@ -58,6 +58,10 @@ To use a self-configured Azure application, you need to add a new Azure App Regi
 * Token - Tenant ID
```

Comment:
need to remove that and refer to https://xsoar.pan.dev/docs/reference/articles/microsoft-integrations---authentication

see what its like in other microsoft integration docs